TA-puppet-alert-actions/bin/ta_puppet_alert_actions/pie/bolt.py

```diff
@@ -25,6 +25,19 @@ def post(endpoint, req, headers):
 
   return json.loads(r.text)['job']
 
+def post_plan(endpoint, req, headers):
+
+  try:
+    r = requests.post(endpoint, json=req, headers=headers, verify=False)
+  except:
+    print('Unexpected error:', sys.exc_info()[0])
```

Comment:
Will this appear in the logs?  Just if the post fails and we are swallowing the exception here, we need it to be clear in the logs is all that the post failed.

Reply:
Forced a failure and confirmed the error is passed up to the Splunk logs:

```
Unable to request Puppet plan before HTTPSConnectionPool(host='none', port=8143): Max retries exceeded with url: /orchestrator/v1/command/plan_run (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f061f6ec5d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
```

I did find a minor typo though in `getplanresult`. Going to push that quick change and merge. 👍